### ISSUE TITLE
Prevent users from using ImageModifier cumulatively

### ISSUE THREAD

**Issue by Hckjs** [cta-observatory/ctapipe#2410]
Currently, you can always reapply the `ImageModifer` to already modified images using the process tool. I think it would be useful to prevent users from doing so for easier reproducibility of the data.

**Comment by kosack** [comment]
I guess technically you have the provenance log for each time you ran, and new files are written as well, so the chain of what happened is recorded.  There could be a use case to apply modifications more than once.   Is the problem that somebody might do this by accident?

**Comment by Hckjs** [comment]
> Is the problem that somebody might do this by accident?

yes, at least a warning might be useful here i think - but in addition i also couldn't think of any use case, where it wouldn't be a better practice to just apply the image modifications on the **not** already modified image. Having just one specific set of applied modification parameters makes it easier (for others) to reproduce and compare the modified data. 

I mean of course you can reproduce it anyway with the provenance logs but i was thinking more of a better/easier practice.
